### ISSUE TITLE
MRTK3 TTS should support configuration options

### ISSUE THREAD

**Issue by david-c-kline** [microsoft/MixedRealityToolkit-Unity#11169]
Things such as:
* read speed
* voice selection
* etc

**Comment by AMollis** [comment]
@srinjoym  Are there missing features from MRTK2?

**Comment by srinjoym** [comment]
This was supported in the MixedReality.Toolkit.Audio package in MRTK 2: https://learn.microsoft.com/en-us/dotnet/api/microsoft.mixedreality.toolkit.audio.texttospeech?view=mixed-reality-toolkit-unity-2020-dotnet-2.8.0. We will need to port the functionality to MRTK3

**Comment by srinjoym** [comment]
Added support for custom voices with https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11458
